query/src/frontend/reorg.rs

```diff
@@ -48,6 +48,29 @@ impl ReorgPlanner {
         Self::default()
     }
 
+    /// Creates an execution plan for a scan operation, which is used primarily
+    /// to load chunks from one storage medium to another.
+    pub fn scan_plan<C, I>(
+        &self,
+        schema: Arc<Schema>,
+        chunks: I,
+    ) -> Result<(Arc<Schema>, LogicalPlan)>
+    where
+        C: QueryChunk + 'static,
+        I: IntoIterator<Item = Arc<C>>,
+    {
```

Comment:
I think this plan will only be correct for a single chunk -- if there are multiple chunks there is a potential for overlap that does need to be considered or else duplicate data will come out.

Given `load_chunk` only has a single chunk, I think you could simplify this plan as well as avoid the potential bug if it were used with multiple chinks


Untested, but something like:

```suggestion
    pub fn scan_plan<C, I>(
        &self,
        schema: Arc<Schema>,
        chunk: Arc<Chunk>,
    ) -> Result<(Arc<Schema>, LogicalPlan)>
    where
        C: QueryChunk + 'static,
    {
```

Reply:
@alamb would you prefer I just build a basic logical plan in the load code here? 

https://github.com/influxdata/influxdb_iox/blob/er/perf/load_chunk_rub/server/src/db/lifecycle/load.rs#L48

I will simplify the plan to a single chunk either way.